website/content/api-docs/system/lease-count-quotas.mdx

```diff
@@ -131,8 +137,10 @@ $ curl \
 
 ## List lease count quotas
 
-This endpoint returns a list of all the lease count quotas. A 404 response will
-be returned if no lease count quota has been created.
+This endpoint returns a list of all the lease count quotas across all namespaces.
+The global visibility differs from the restricted access enforced when modifying
```

Comment:
Might be my brain but this one was a hard for me to absorb on first glance. Maybe reword this a bit?

Reply:
I agree, thanks for the feedback. I took a stab at rewording this in https://github.com/hashicorp/vault/pull/25894/commits/2c747d7c58d9ab3db3e0f78a8f76869c489ff3e9.